invenio_app_rdm/theme/assets/js/invenio_app_rdm/search/config/index.js

```diff
@@ -0,0 +1,68 @@
+export { templates } from "./templates";
+
+const aggregations = [
+  {
+    title: "Access Right",
+    agg: {
+      field: "access_right",
+      aggName: "access_right"
+    }
+  },
+  {
+    title: "Types",
+    agg: {
+      field: "resource_type",
+      aggName: "resource_type"
+    }
+  }
+];
+
+const sortValues = [
+  {
+    text: "Best match",
+    sortBy: "bestmatch",
+    sortOrder: "desc",
+    defaultOnEmptyString: true
+  },
+  {
+    text: "Newest",
+    sortBy: "mostrecent",
+    sortOrder: "asc",
+    default: true
+  },
+  {
+    text: "Oldest",
+    sortBy: "mostrecent",
+    sortOrder: "desc"
+  }
+];
+
+const resultsPerPageValues = [
+  {
+    text: "10",
+    value: 10
+  },
+  {
+    text: "20",
+    value: 20
+  },
+  {
+    text: "50",
+    value: 50
+  }
+];
+
+const logo = "/static/images/logo.svg";
+
+const searchApi = {
+  url: "https://localhost:5000/api/records/",
+  timeout: 5000
+};
+
+export const config = {
+  logo,
+  searchApi,
+  aggregations,
+  sortValues,
+  resultsPerPageValues
+};
```

Comment:
If we can, how do we override these from the application's configuration (`config.py`)? `localhost:5000` is bound to require a change right?

Reply:
The whole config mechanism and how it is communicated from the backend is not part of this PR. I introduced the `config.js` just to start having one central place that my components are getting parametrized values and then this module will be the responsible to get the configuration from the backend.

Reply:
Got it. Made the issue for it: https://github.com/inveniosoftware/invenio-app-rdm/issues/82